chore(docs): weekly review (#4088)
scaleway/docs-content: storage/block/api-cli/unmounting-a-volume.mdx

## Unmounting a Block Storage volume

> Unmounting a volume means detaching it from the operating system so it is no longer accessible. Unmouting a volume is useful if you want to stop using it, prevent data corruption, or if you plan to [detach it](/storage/block/how-to/detach-a-volume/) from your Instance, and attach it to another. This page shows you how to unmount a volume from the operating system using the `umount` command.

Reviewer: ```suggestion
Unmounting a volume means detaching it from the operating system so it is no longer accessible. You can do this if you want to stop using it, prevent data corruption, or plan to [detach it](/storage/block/how-to/detach-a-volume/) from your Instance and attach it to another. This page shows you how to unmount a volume from the operating system using the `umount` command.
```
